lib/getLatestRelease.js

```diff
@@ -20,7 +20,7 @@ const getOS = str => {
     return 'macOS';
   } else if (includes(str, 'win32') || includes(str, '.exe')) {
     return 'Windows';
-  } else if (includes(str, 'linux')) {
+  } else if (includes(str, 'linux') || includes(str, 'appimage')) {
```

Comment:
is this caSe-iNsesiTiVe? ;-)  (or if this is always lower-case, perhaps you could add a comment?)

Reply:
Yup, it's lower-cased in `prepAssets()` below